From a2193ad435fdb293fc9829ac05dfe38e53b7e097 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 29 Nov 2012 09:47:24 -0800
Subject: [PATCH] gpio: xilinx_gpiops: Remove unnecessary #include

linux/errno.h is not required by the driver and thus removed
from the included files.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
---
 drivers/gpio/gpio-xilinxps.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/gpio/gpio-xilinxps.c b/drivers/gpio/gpio-xilinxps.c
index e97eb11..aef0618 100644
--- a/drivers/gpio/gpio-xilinxps.c
+++ b/drivers/gpio/gpio-xilinxps.c
@@ -15,7 +15,6 @@
 
 #include <linux/clk.h>
 #include <linux/err.h>
-#include <linux/errno.h>
 #include <linux/export.h>
 #include <linux/gpio.h>
 #include <linux/init.h>
-- 
1.7.0.4

